TLM/TLM/U/Autosize/UPadding.cs

```diff
@@ -0,0 +1,38 @@
+namespace TrafficManager.U.Autosize {
```

Comment:
is there a reason you avoided using `UnityEngine.Rect`?

Reply:
Different semantic. Rect has x, y, width, height. And padding has left, right, top, bottom.